app/components/m-header.hbs

```diff
@@ -1,87 +1,168 @@
-<div class="auk-navbar auk-navbar--bordered-bottom auk-navbar--no-pad auk-navbar--auto l-main-header"
-     id="c-main-nav">
```

Comment:
Zoals Karel aangeeft, die auk-navbar div mist nu, de zou moeten terugkomen.
Die div zit ook in de prototype

`<div class="auk-navbar auk-navbar--bordered-bottom auk-navbar--white auk-navbar--no-pad">`